fix: respect multipart form-data evaluation (#2229)
Redocly/redocly-cli: .changeset/hot-goats-cover.md

---
"@redocly/respect-core": patch
"@redocly/cli": patch
---

> Fix: ensure multipart form-data parameters are properly resolved and evaluated before sending requests.

Reviewer: ```suggestion
Fixed an issue where multipart form-data parameters were not properly resolved and evaluated before sending requests.
```
